### PR TITLE
Start testing pypy3

### PR DESCRIPTION
**Pull request by jcpunk**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by alimcmaster1** [comment]
Thanks for taking a look, our preference is to run more of our CI on Azure compared to Travis. Is it easy enough to move over? (I would wait for some other maintainers to validate this before making any changes)

Ref: Email from @bashtage this week.

File: ci/setup_env.sh [hunk 1]
```diff
@@ -61,9 +63,15 @@ which conda
 echo
 echo "update conda"
 conda config --set ssl_verify false
-conda config --set quiet true --set always_yes true --set changeps1 false
-conda install pip conda  # create conda to create a historical artifact for pip & setuptools
-conda update -n base conda
+conda config --set quiet true --set always_yes true --set changeps1 false --set channel_priority strict
+if [ "$JOB" == "pypy3" ]; then
+    conda create -n pypy3 pypy
```
Comment (jreback) [inline ci/setup_env.sh:68]: you shouldn't need to update this, what is the reason?

**Review by jreback — CHANGES_REQUESTED** [review]
yeah we would prefer to do this on azure. config should be not much to change.

File: ci/setup_env.sh (hunk 1 above)
Comment (jcpunk) [reply ci/setup_env.sh:68]: In my test container, when I didn't do this, conda tried to use CPython rather than pypy3.

**Comment by jcpunk** [comment]
I'll confess I'm unsure how to provide a PR for the azure bits....

**Comment by WillAyd** [comment]
@jcpunk might help to look at the yaml docs for pipelines:

https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema

**Comment by jcpunk** [comment]
now that I've chased down the azure bits and gotten the manylinux wheels somewhat sorted, looks like the manylinux containers aren't ready for pypy yet.

I'll close this out for now and revisit when the manylinux container has pypy support.  No sense in hacking it in now.

**Comment by mattip** [comment]
Manylinux support pypy3.7 and pypy3.8. They are installed in manylinux2014:latest (from 5 days ago). For instance, pypy3.8 is in /opt/python/pp38-pypy38_pp73

**Pull request closed without merging**